frontend/kesaseteli/employer/src/components/application/summary/EmploymentFieldSummary.sc.ts

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const $EmploymentFieldSummary = styled.div`
```

Comment:
`styled($GridCell)` and perhaps some themed font size here too?

Reply:
Fixed :+1: